client/merchant-feedback-prompt/positive-modal.tsx

```diff
@@ -46,7 +46,7 @@ export const PositiveFeedbackModal: React.FC< PositiveFeedbackModalProps > = ( {
 		>
 			<p>
 				{ __(
-					'Thanks for sharing your feedback on WooPayments! Would you mind leaving us a quick review on WordPress.org?',
+					'Thanks for sharing your feedback on WooPayments! Would you mind leaving us a 5-star rating and quick review on WordPress.org?',
```

Comment:
Nit - add an 'a' before 'review'.

```suggestion
					'Thanks for sharing your feedback on WooPayments! Would you mind leaving us a 5-star rating and a quick review on WordPress.org?',
```

Reply:
Thanks @nagpai. I considered that and wasn't sure if "5-star rating and quick review" was a single "bundle", but your suggestion locks it in.